tickets.php

```diff
@@ -19,6 +19,35 @@
 
 if ($thisclient->isGuest())
     $_REQUEST['id'] = $thisclient->getTicketId();
+function closetags($html) {

```

Comment:
basically a function to close open tags in messages. such as divs etc. 
use tidy if its available, else use the regex